src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
@weshaggard how does a package get the -beta-* suffix? Is there a file we need to modify to mark our packages as beta? Or is it all automagically done?